kolibri/utils/conf.py

```diff
@@ -54,7 +48,8 @@
     "kolibri.plugins.video_mp4_render",
     "kolibri.plugins.audio_mp3_render",
     "kolibri.plugins.setup_wizard",
-    "kolibri.plugins.coach_tools"
+    "kolibri.plugins.coach_tools",
+    "exercise_perseus_render"
```

Comment:
`render` looks funny to me too, but note the names of the audio/video ones above, too